services/api/routers/crud/crud.py

```diff
@@ -42,7 +42,7 @@ def get_closest_station(zipcode: str, db: Session):
     return result
 
 
-def get_data(station: str, db: Session):
+def get_data(ids: list[str], db: Session):
     stmt = text(
         """
```

Comment:
I feel like we should probs be using the orm for these crud functions right? nbd for now, I think a lot of this stuff will be reworked soon